lib/mode-reset/index.js

```diff
@@ -294,12 +305,14 @@ ResetPanel.prototype.submit = function () {
       email_input.val('');
       widget._signinPanel(panel.options);
       widget._showSuccess(widget.options.i18n.t('reset:successText'));
-
+      widget.emit('reset success');
       return 'function' === typeof callback ? callback.apply(widget, args) : null;
     }
 
     widget.setPanel(panel);
 
+    widget.emit('reset error', err);
```

Comment:
here we are triggering the error before the setPanel which will trigger the error after the ready...

different from what we established, right?